tests/e2e.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash -ex
+
+cd "$(dirname "$0")"
+cd ..
+
+# Prevent postfix ipv6 errors
+sed -i "/^inet_protocols/c\inet_protocols = ipv4" /etc/postfix/main.cf
+
+# start postfix
+newaliases
+postfix start || postfix reload
+
+# Install zeyple using deb packages from previous step
+dpkg -i ./fpm/zeyple.deb
+cp /usr/local/bin/zeyple.conf.example /etc/zeyple.conf
+
+# Import GPG key
+gpg --homedir /var/lib/zeyple/keys --import ./tests/e2e_keys.gpg
+chown -R zeyple: /var/lib/zeyple/keys
```

Comment:
This `chown` shouldn't be necessary if you add `sudo -u zeyple ...` to the previous `gpg` command like described in https://github.com/infertux/zeyple#key-management

Reply:
Fixed, thanks